### PR TITLE
[KHRGA-39] Updated EP id classes

### PR DESCRIPTION
**Pull request by imvucic**
Issue: KHRGA-39

### CODE REVIEW

File: source/iface/id.rst [hunk 1]
```diff
@@ -12,13 +12,14 @@
...
 
-   template <int dimensions = 1>
+   template <int Dimensions = 1>
    class id;
 
-The ``id`` is an abstraction that describes the location of a point in
-a :ref:`range`. Examples includes use as an index in an
-:ref:`command-accessor` and as an argument to a kernel function in a
-:ref:`handler-parallel_for` to identify the work item.
+The ``id<int Dimensions>`` is a vector of Dimensions that is used
```
Comment (mkinsner) [inline source/iface/id.rst:18]:
```suggestion
The ``id`` class is a collection of size ``Dimensions`` that is used
```

File: source/iface/id.rst [hunk 2]
```diff
@@ -28,51 +29,167 @@ a :ref:`range`. Examples includes use as an index in an
...
+
 ``get``
 =======
 
 ::
 
-  size_t get(int dimension) const;
+  size_t get(int dimension) const
+
+Return the value of the ``id`` for dimension ``Dimension``.
```
Comment (mkinsner) [inline source/iface/id.rst:60]:
```suggestion
Return the value of the ``id`` for dimension ``dimension``.
```

File: source/iface/id.rst [hunk 3]
```diff
@@ -28,51 +29,167 @@ a :ref:`range`. Examples includes use as an index in an
...
+
+Where ``OP`` is: ``+``, ``-``, ``*``, ``/``, ``%``, ``<<``,
+``>>``, ``&``, ``|``, ``^``, ``&&``, ``||``, ``<``, ``>``,
+``<=``, ``>=``.
+
+Constructs and returns a new instance of the SYCL ``id`` class template
+with the same dimensionality as the ``rhs`` SYCL ``id``, where each
+element of the new SYCL ``id`` instance is the result of an element-wise
+``OP`` operator between the ``lhs size_t`` and each element of the
+``rhs`` SYCL ``ange``. If the operator returns a ``bool``,
```
Comment (mkinsner) [inline source/iface/id.rst:163]:
```suggestion
``rhs`` SYCL ``range``. If the operator returns a ``bool``,
```

File: source/iface/id.rst (hunk 1 above)
Comment (imvucic) [reply source/iface/id.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/111/commits/3a9bce906a9622857614ce6806e543f7958ef6c8

File: source/iface/id.rst (hunk 2 above)
Comment (imvucic) [reply source/iface/id.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/111/commits/3a9bce906a9622857614ce6806e543f7958ef6c8

File: source/iface/id.rst (hunk 3 above)
Comment (imvucic) [reply source/iface/id.rst]: Fixed with : https://github.com/KhronosGroup/SYCL_Reference/pull/111/commits/3a9bce906a9622857614ce6806e543f7958ef6c8

**Review by mkinsner — APPROVED** [review]

**Pull request merged**